chronos/asyncloop.nim

```diff
@@ -803,6 +805,31 @@ proc sleepAsync*(ms: int): Future[void] {.
      inline, deprecated: "Use sleepAsync(Duration)".} =
   result = sleepAsync(ms.milliseconds())
 
+proc stepsAsync*(number: int): Future[void] =
```

Comment:
aka `yieldAsync`?

Reply:
Actually not, because `yieldAsync` should not introduce 100% CPU consumption, this procedure could easily achieve it.

Reply:
So the purpose of this primitive is to help creating complex tests...

Reply:
ok, that could be documented here - also, there's no such thing as "for testing", if it's there, it should be generally supported - ie this would be useful for pausing the queue processing in nbc and letting networking do its work

Reply:
At any case this primitive MUST not be used for pausing the queue processing and let networking do its work.